toc.md

```diff
@@ -812,12 +815,22 @@ In order to get around the asynchronous nature of threads, instead of asserting
 #### Workerholic compared to the Gold Standard: Sidekiq
 ![benchmark_workerholic_sidekiq](/images/benchmark_workerholic_sidekiq.png)
 
-Finally, we wanted to compare with Sidekiq one last time with each types of jobs individually. We're on par with Sidekiq, and only slightly faster than Sidekiq each time, and as we mentioned before this is because Sidekiq is a more mature and robust solution with many more features and handles more use cases.
+Finally, we wanted to compare with Sidekiq one last time with each types of jobs individually. We're on par with Sidekiq, and only slightly faster than Sidekiq each time, and as we mentioned before this is because Sidekiq is a more mature and robust solution with a much larger feature-set.
 
 #### JRuby
 ![benchmark_jruby](/images/benchmark_jruby.png)
 
-We also decided to compare the results of jRuby vs MRI. Because jRuby can run in parallel without the need of spinning up multiple processes, we found that CPU blocking jobs were much faster in jRuby than in MRI, which is what we would expect.
+We also decided to compare the results of JRuby vs MRI. Because jRuby can run in parallel without the need of spinning up multiple processes, we found that execution of CPU blocking jobs was much faster in JRuby than in MRI, which is what we would expect.
 
 ## Conclusion
-N/A.
+
+It was a lot of fun to work on this project. We learned a lot about threaded code, memory usage by different types of OS entities, Redis data structures and other Ruby interpreters. We really hope our findings will help other developers in their journey through the pathways of threaded code.
```

Comment:
I wouldn't emphasize the threading part. I'd talk about the fact that with this blog post we hope to share some useful knowledge about how to build a BJP from scratch and also that for those not planning on building one, at least exposing the process of doing so with Workerholic will shed some light on how BJPs work. Then as a secondary point we can talk about what we learned without so much emphasis on such low level concepts, for example: 
> We learned and deepened our knowledge about:
> -  concurrency, and specifically enabling concurrency using threads
> -  parallelism, and specifically enabling parallelism using multiple processes
> - lower level details about threads and processes, such as their memory pattern
> - the impact on computational and memory resources when using multiple threads and multiple processes
> - using Redis as a key:value store and taking advantage of its awesome features such as using specific data structures and taking DB snapshots
> - building a library in the ruby eco-system
> - providing users with an easy to use and efficient CLI